sim/dex.js

```diff
@@ -1411,6 +1411,7 @@ class ModdedDex {
 			if (!format.column) format.column = column;
 			if (this.formatsCache[id]) throw new Error(`Format #${i + 1} has a duplicate ID: '${id}'`);
 			format.effectType = 'Format';
+			format.baseRuleset = format.ruleset ? format.ruleset.slice() : [];
```

Comment:
It is, but the ruleset may already be altered when <code>format</code> is passed to the Format constructor because formats aren't cached like the other data types.